src/accountImporter.js

```diff
@@ -175,14 +175,15 @@ var _validateRequiredParameters = function(options) {
         );
       }
       return { hashAlgo: hashAlgo, hashKey: options.hashKey, valid: true };
-    case "MD5":
     case "SHA1":
     case "SHA256":
     case "SHA512":
+      // MD5 is [0,8192] but SHA1, SHA256, and SHA512 are [1,8192]
```

Comment:
You removed MD5 from the cases above. I don't think MD5 logic will run here.

Reply:
Phew that was close.